inference-engine/tests/functional/shared_test_classes/src/single_layer/convolution.cpp

```diff
@@ -41,29 +46,46 @@ std::string ConvolutionLayerTest::getTestCaseName(const testing::TestParamInfo<c
 
 void ConvolutionLayerTest::SetUp() {
     convSpecificParams convParams;
-    std::vector<size_t> inputShape;
-    auto netPrecision   = InferenceEngine::Precision::UNSPECIFIED;
-    std::tie(convParams, netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, targetDevice) =
+    std::vector<std::vector<std::pair<size_t, size_t>>> inputShape;
+    std::vector<std::vector<InferenceEngine::SizeVector>> targetShapes;
+    std::tie(convParams, netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, targetShapes, targetDevice) =
         this->GetParam();
-    ngraph::op::PadType padType;
-    InferenceEngine::SizeVector kernel, stride, dilation;
-    std::vector<ptrdiff_t> padBegin, padEnd;
-    size_t convOutChannels;
+    for (auto&& targetShape : targetShapes) {
+        targetStaticShapes.emplace_back(
+                std::vector<ngraph::Shape>{ngraph::Shape{targetShape.front()}, ngraph::Shape{targetShape.front()}});
+    }
+    inputDynamicShape.emplace_back(
+            FuncTestUtils::PartialShapeUtils::vec2partialshape(
+                    inputShape.empty() ?
+                    std::vector<std::pair<size_t, size_t>>{} :
```

Comment:
One important question - why we cann't use 'ngraph::PartialShape' as a parameter of tests. As for me it will be more clear

Reply:
What do you mean using `google test does not know ngraph::PartialShape eventually`?

Reply:
Sorry if I make you misunderstand, it meant, as I know, google test does not recognize ngraph::PartialShape and it may need works to recognize it and I have no idea of it.

Reply:
ok, got it

Reply:
![image](https://user-images.githubusercontent.com/22008923/135260977-3047c44c-99ba-4c75-8072-e9078cf75a95.png)